tests/suites/5_ee/01_vacuum/01_0002_ee_vacuum_drop_table.sh

```diff
@@ -119,7 +119,7 @@ echo "select * from test_vacuum_drop_4.b"  | $BENDSQL_CLIENT_CONNECT
 
 ## test vacuum table output
 echo "create table test_vacuum_drop_4.c(c int)" | $BENDSQL_CLIENT_CONNECT
-echo "INSERT INTO test_vacuum_drop_4.c VALUES (1)" | $BENDSQL_CLIENT_OUTPUT_NULL
+echo "INSERT INTO test_vacuum_drop_4.c VALUES (1),(2)" | $BENDSQL_CLIENT_OUTPUT_NULL
```

Comment:
Failed to understand the purpose of this adjustment, if we don’t insert the extra line here, will this test case fail?